packages/sources/src/addSourceWizard/hardcodedComponents/openshift/costManagement.js

```diff
@@ -19,25 +18,17 @@ export const ConfigureCostOperator = () => (
                     for <i>cost management</i> ).
                 </TextListItem>
                 <TextListItem component='li'>
-                    When configuration is complete provide your cluster identifier.<br />
-                    Enter the cluster identifier below:
+                    When configuration is complete, enter the cluster identifier below.<br />
+                    The cluster identifier can be found in the cluster&apos;s Help &gt; About screen
                 </TextListItem>
             </TextList>
         </TextContent>
     </TextContent>
 );
 
 export const ClusterIdentifierLabel = () => (
```

Comment:
you don’t need this anymore. Just set `label` and `isRequired` in the schema.

Reply:
Done